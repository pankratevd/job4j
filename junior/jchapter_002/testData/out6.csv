Всего групп: 6
Группа 1
"8";"527";"539"
"8";"427";""
"17";"121";"539"

Группа 2
"1";"22";"31"
"3";"23";"31"
"1";"21";"31"

Группа 3
"4";"24";"34"
"4";"25";"35"

Группа 4
"6";"26";"36"
"5";"26";"36"

Группа 5
"117";"";"37"
"7";"27";"37"

Группа 6
"9";"28";"637"
"10";"28";"647"

