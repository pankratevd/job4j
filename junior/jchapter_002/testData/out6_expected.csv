Всего групп: 6
Группа 1
"8";"427";""
"8";"527";"539"
"17";"121";"539"

Группа 2
"1";"22";"31"
"3";"23";"31"
"1";"21";"31"

Группа 3
"117";"";"37"
"7";"27";"37"

Группа 4
"9";"28";"637"
"10";"28";"647"

Группа 5
"4";"24";"34"
"4";"25";"35"

Группа 6
"5";"26";"36"
"6";"26";"36"

